smartplaylist: Enrich original "Creating output" with track counts; CLI output overhaul (#6493)
beetbox/beets: docs/changelog.rst

      after upgrading to trigger the migration. Only then you can safely move
      the library to a new location.

- :doc:`plugins/smartplaylist`: A ``--quiet`` flag is added to the ``splupdate``
  command to suppress several log messages for each updated playlist and in
  pretend mode only prints the playlist name and the number of tracks instead of
  each track name. This can be useful when testing out new entries in the

>   configuration.

Author: Same reasoning as pointed out here: https://github.com/beetbox/beets/pull/6493/changes#r3084145648

- Metadata source plugin searches and lookups are now executed concurrently,
  speeding up lookups when multiple plugins (e.g. MusicBrainz and Spotify) are
  enabled.
- :doc:`plugins/smartplaylist`: The ``splupdate`` command output is
  restructured. The per-playlist summary now includes a track count (``Creating
  playlist X: N tracks.``) at ``INFO`` level. Per-track details are shown only
  in ``DEBUG`` level. The ``--pretend`` flag produces the same output but
  reports *"N playlists would be updated"* instead of *"N playlists updated"*.

>   The ``--format`` option allows customizing the track line format. :bug:`6183`

Reviewer: User don't know anything about logging levels. We should replace __`INFO` level__ with __by default__ and __`DEBUG` level__ with __using `-vv` flag__.

And mention that the `--format` option is new!

Reviewer: And also mention that the `--pretend-paths` option is removed, suggesting users to use `--format='$path'`

Author: Fixed the changelog

- Metadata source plugin searches and lookups are now executed concurrently,
  speeding up lookups when multiple plugins (e.g. MusicBrainz and Spotify) are
  enabled.
- :doc:`plugins/smartplaylist`: The ``splupdate`` command output is
  restructured. The per-playlist summary now includes a track count. Per-track

>   details are shown only in ``DEBUG`` log level (``beet -v splupdate``). The

Reviewer: ```suggestion
  details are shown only when ``-v`` flag is provided (``beet -v splupdate``). The
```

  ``--pretend`` flag produces the same output but reports *"N playlists would be
  updated"* instead of *"N playlists updated"*. The ``--format`` option allows
  customizing the track line format. The ``--pretend-paths`` option was removed

>   since the per-track log line is now configurable. :bug:`6183`

Reviewer: ```suggestion
  (use ``--format='$path'`` instead). :bug:`6183`
```
